SensusService/Protocol.cs

```diff
@@ -387,6 +392,10 @@ public static void RunUnitTestingProtocol(Stream unitTestingProtocolFile)
         private bool _shareable;
         private List<PointOfInterest> _pointsOfInterest;
         private string _description;
+        private DateTime _startTime;
+        private bool _startImmediately;
+        private DateTime _endTime;
+        private bool _continueIndefinitely;
```

Comment:
Sorry, Wes's stuff. I didn't realize some o f his changes had been mixed in with mine.